### ISSUE TITLE
Ability to compare 2 packages, not just 2 versions

### ISSUE THREAD

**Issue by adamralph** [ParticularLabs/APIComparer#37]
OK, here's a curve ball... :wink:

Following the pattern of `xunit` 2.0, `Xbehave` 2.0 is now a meta-package containing nothing but a dependency on `Xbehave.Core` and `xunit`, where `Xbehave.Core` is the package containing the actual assemblies. The reason for this is that `xunit` is a meta-package containing deps on `xunit.core` and `xunit.assert`, allowing users to install just `xunit.core` when they want to use another assertion library. Similarly, users can just install `Xbehave.Core` which only takes a dep on `xunit.core`.

So the bottom line is that I'd like to be able to compare `Xbehave` 1.1.0 with `Xbehave.Core` 2.0.0-rc002-build543.

I can hear the groans already :stuck_out_tongue_closed_eyes:.

**Comment by andreasohlund** [comment]
We have similar needs since nsb.interfaces and nsb.core v4 is replaced by nsb.core only in v5

**Comment by SimonCropp** [comment]
yeah when i created this http://docs.particular.net/nservicebus/upgrades/4to5diff i fudged it.

bottom line is we need metadata in the nuget to achieve this

**Comment by adamralph** [comment]
Ah, great. So not such an edge case then...
